### PR TITLE
abe - document createrepo

### PR DESCRIPTION
**Pull request by adean20csc**
Documenting the createrepo role.
No testing required.

### CODE REVIEW

**Comment by jcpowermac** [comment]
Review :+1:

File: ansible/roles/createrepo/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  * Create an offline repo file in the repository that can be distributed to any hosts /etc/yum.conf directory.
+
+  * Create an offline repo config for apache and puts it in the /etc/httpd/conf.d directory.
+
+  * Restart the httpd service so the changes take effect.
+
+Requirements
+------------
+
+The role requires a pre-installed version of RHEL on the dcaf-abe autodeploy node.
```
Comment (jdextercsc) [inline ansible/roles/createrepo/README.md]: I thought this downloaded the rpms needed?  or is this saying that it depends on that being done earlier in the playbook?

Comment (mtnbikenc) [inline ansible/roles/createrepo/README.md]: The needed RPMs are downloaded during stage_resources.  When this role is run, the autodeploy node is offline.  This role just sets up the config necessary to run the local_repo for the other nodes to install from.

**Comment by mtnbikenc** [comment]
Review :+1:

**Pull request merged**